aspnetcore/signalr/configuration.md

```diff
@@ -136,6 +137,21 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
 The following table describes options for configuring ASP.NET Core SignalR's advanced HTTP options:
 
+::: moniker range=">= aspnetcore-3.0"
+
+| Option | Default Value | Description |
+| ------ | ------------- | ----------- |
+| `ApplicationMaxBufferSize` | 32 KB | The maximum number of bytes received from the client that the server buffers before applying backpressure. Increasing this value allows the server to receive larger messages more quickly without applying backpressure, but can negatively impact memory consumption. |
+| `AuthorizationData` | Data automatically gathered from the `Authorize` attributes applied to the Hub class. | A list of [IAuthorizeData](/dotnet/api/microsoft.aspnetcore.authorization.iauthorizedata) objects used to determine if a client is authorized to connect to the hub. |
+| `TransportMaxBufferSize` | 32 KB | The maximum number of bytes sent by the app that the server buffers before applying backpressure. Increasing this value allows the server to send larger messages more quickly without applying backpressure, but can negatively impact memory consumption. |
+| `Transports` | All Transports are enabled. | A bitmask of `HttpTransportType` values that can restrict the transports a client can use to connect. |
```

Comment:
Nit: I think the following might be more clear to your average C# dev.

```suggestion
| `Transports` | All Transports are enabled. | A bit flags enum of `HttpTransportType` values that can restrict the transports a client can use to connect. |
```

Reply:
There's another table that also uses `bitmask`. @BrennanConroy you should update that one too.